dll/win32/setupapi/devinst.c

```diff
@@ -2704,22 +2748,22 @@ HKEY WINAPI SetupDiCreateDeviceInterfaceRegKeyW(
                                             InfHandle,
```

Comment:
(Side-note: this SetupInstallFromInfSection call is a SetupInstallFromInfSectionW one)